### ISSUE TITLE
`ERROR` verdict for both branches dead in SV-COMP

### ISSUE THREAD

**Issue by sim642** [goblint/analyzer#1576]
Apparently there are SV-COMP tasks where we output the "both branches dead" unsoundness warning, but still happily give a `true` SV-COMP result. We've been lucky that this hasn't caused unsound verdicts.
In these cases the verdict should instead be something like `ERROR (both branches dead)`.

We do this for fixpoint errors with `ERROR (fixpoint)`.

**Comment by karoliineh** [comment]
The example I stumbled upon was `termination-restricted-15/IntPath.c`
```c
extern int __VERIFIER_nondet_int(void);

int main() {
    int i;
    int x;
    int y;
    i = __VERIFIER_nondet_int();
    x = 0;
    y = 0;
    
    if (i > 10) {
        x = 1;
    } else {
        y = 1;
    }
    while (x == y) { }
    
    return 0;
}
```
which yields:
```
...
[Error][Analyzer][Unsound] both branches over condition 'x == y' are dead (./termination-restricted-15/IntPath.c:18:12-18:18)
SV-COMP result: true
[Warning][Analyzer] Invariant Apron: cannot convert to cil var in overflow preserving manner: term18_5_id0#1998
[Info][Witness] witness generation summary:
  total generation entries: 1
[Error][Analyzer][Unsound] Both branches dead
```

**Comment by michael-schwarz** [comment]
> The example I stumbled upon was `termination-restricted-15/IntPath.c`

Looks like it may be an unsoundness in our apron bindings we only bump into for polyhedra, which are enabled for termination.

**Comment by sim642** [comment]
> Looks like it may be an unsoundness in our apron bindings we only bump into for polyhedra, which are enabled for termination.

It's not that, because it also happens with octagon. Anyway, I extracted the unsoundness itself to #1578.

**Comment by sim642** [comment]
There's something else odd going on as well: even though termination is supposed to make us use polyhedra, it's actually still using octagon. Explicitly setting the domain from command line works.
